worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -184,6 +184,16 @@ function fieldExists(document, fieldName) {
     return document.getField(fieldName).hasValues();
 }
 
+function isNotEmptyMap(obj) {   
+    if(!obj)
+        return false;
+    var jsonObj = JSON.parse(obj);
```

Comment:
Is that caused you change the double quotes in darwin-worker-workflow?
You need to be sure that's safe for the rest of the product. Up until now people could used "